src/tasks/task3.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-nested-ternary */
+const namesObjectData = {
+  QUANTITY: 'quantity',
+  PRICE: 'price',
+  PRICE_FOR_PAIR: 'priceForPair',
+};
+
+module.exports = arrayClothes => {
+  const result = arrayClothes.map(clothes => {
+    Object.values(namesObjectData).forEach(type => {
+      if (!(type in clothes)) {
+        return {
+          ...clothes,
+          [type]:
+            type === namesObjectData.PRICE
+              ? clothes[namesObjectData.PRICE_FOR_PAIR]
+              : type === namesObjectData.PRICE_FOR_PAIR
+              ? `$${clothes[namesObjectData.PRICE].split('$')[1] * 2}`
+              : 0,
+        };
```

Comment:
How about this?
```
const ( type, color, quantity } = clothes;
return {
  type: type || 'N/A',
  color: color || 'N/A',
  quantity: quantity || 0,
}
```
Just add `price` and `priceForPair` fields using this logic.
This way it looks much better and easier and `namesObjectData` is not needed anymore.